.github/workflows/node-tests.yml

```diff
@@ -37,12 +37,13 @@ on:
         type: "string"
 
 jobs:
-  npm-ci:
+  node-tests:
```

Comment:
I think this will change the required actions from other projects.

Reply:
Oh boy.